test/parallel/test-child-process-exec-abortcontroller-promisified.js

```diff
@@ -18,11 +18,35 @@ const waitCommand = common.isWindows ?
   const ac = new AbortController();
   const signal = ac.signal;
   const promise = execPromisifed(waitCommand, { signal });
-  assert.rejects(promise, /AbortError/, 'post aborted sync signal failed')
-        .then(common.mustCall());
+  assert.rejects(promise, {
+    name: 'AbortError'
```

Comment:
nit: Should we also test that the `cause` property is set when `ac.abort()` is called with no parameters or `undefined`?
```suggestion
    name: 'AbortError',
    cause: signal.reason,
```

Reply:
Understood, updating!

Reply:
updated!